test/browser_tests/spec_suites/organisms/pagination.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var Blog = require(
+    '../../page_objects/page_blog.js'
+  );
+
+describe( 'Pagination', function() {
+  var page;
+
+  beforeEach( function() {
+    page = new Blog();
+    page.get();
+  } );
+
+  it( 'should navigate to the second page', function() {
+    page.paginationNextBtn.click()
+    browser.sleep( 1000 );
```

Comment:
@sebworks I tried something similar to check the url on load, then wait for it to not match, but I couldn't get it to work. Any ideas?

```
  it( 'should navigate to the second page', function() {
    var currentUrl = browser.getCurrentUrl();
    page.paginationNextBtn.click()

    browser.wait( function() {
      return browser.getCurrentUrl().then(function( url ) {
        return ( currentUrl !== url );
      } );
    } );

    expect( browser.getCurrentUrl() ).toContain( 'page=2' );
  } );
```